pixie/vm/stdlib.py

```diff
@@ -774,8 +774,23 @@ def _str(self):
     assert isinstance(self, RuntimeException)
     return rt.wrap(self.__repr__())
 
+@extend(_seq, RuntimeException)
+def _seq(self):
+    import pixie.vm.persistent_vector as vector
+    assert isinstance(self, RuntimeException)
+    trace = vector.EMPTY
+    trace = rt.conj(trace, self._data)
+    for x in self._trace:
+        trace = rt.conj(trace, rt.wrap(x.__repr__()))
```

Comment:
I like this, but I'd also like a bit more detail. Can we add a method to each of the ErrorInfo types (found here: https://github.com/pixie-lang/pixie/blob/master/pixie/vm/object.py#L157), that construct a map or something for each frame? As it stands we'd need to parse the strings if we want anything useful out of the trace. So you could probably just clone each `__repr__` method and construct a map instead of a string. So polymorphic code info would look something like `{:type :polymorphic :name "-count" :type <type int>}`